resources/assets/js/components/PostComponent.vue

```diff
@@ -212,7 +212,7 @@
 
     <div v-if="profileLayout == 'moment'" class="momentui">
       <div class="bg-dark mt-md-n4">
-        <div class="container">
+        <div class="container" v-on:doubletap="likeStatus(status, $event)">
```

Comment:
You can remove the first argument from ```likeStatus``` since the status is already known in this context.